mysql-test/suite/innodb/r/log_encrypt_2_mk.result

```diff
@@ -63,9 +63,6 @@ c1	c2
 7	hhhhh
 8	iiiii
 9	jjjjj
-SELECT rotate_system_key("percona_redo");
-rotate_system_key("percona_redo")
-1
 ALTER INSTANCE ROTATE INNODB MASTER KEY;
```

Comment:
why is this missing?

Reply:
Previously it was possible to rotate percona_redo key, even if it didn't exist. Now it is impossible - so performing percona_redo key rotation in case of Master Key encryption does not make sense.